src/main/java/core/basesyntax/service/impl/SupplyOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperations;
+
+public class SupplyOperation implements FruitOperations {
```

Comment:
You can add supply and return operations in ome implementation class. No need to duplicate code.
just make one AddOperationImpl or something like this.

Reply:
done